manifests/u/UbiquitiInc/UI/0.50.1.2/UbiquitiInc.UI.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: UbiquitiInc.UI
+PackageVersion: 0.50.1.2
+PackageLocale: en-US
+Publisher: Ubiquiti Inc
+PackageName: UI
+License: Copyright (c) Ubiquiti Inc. All rights reserved.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate